src/components/utils/file.js

```diff
@@ -1,16 +1,23 @@
 import loadImage from 'blueimp-load-image/js/load-image'
 import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
 
-export const fileToBase64 = (file, callback, errorCallback) => {
+export const convertFile = (file, callback, errorCallback) => {
   const reader = new FileReader();
-  reader.readAsDataURL(file);
-  reader.onload = function () {
-   callback(reader.result)
+  // If PDF return base64. For other images return blob
```

Comment:
Why are PDFs treated differently? Does the client need to handle them differently? If so is it documented?